ote/src/cljs/ote/app/controller/front_page.cljs

```diff
@@ -32,6 +32,16 @@
                          :before-unload-message
                          :navigation-confirm))))
 
+(defn get-transport-operator-data [app]
+   (if (get app :transport-operator-data-loaded? true)
+     (do
+       (comm/post! "transport-operator/data" {}
+                  {:on-success (tuck/send-async! ->TransportOperatorDataResponse)
+                   :on-failure (tuck/send-async! ->TransportOperatorDataFailed)})
+      (assoc app :transport-operator-data-loaded? false
+                 :services-changed? false)
+       ) app))
```

Comment:
confusing paren placement, put closing paren on previous line, so that `app` as the else clause is clear